change/@ni-nimble-components-078b375d-daec-46bc-8525-995725df1168.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Update version of FAST",
+  "packageName": "@ni/nimble-components",
+  "email": "7282195+m-akinc@users.noreply.github.com",
+  "dependentChangeType": "patch"
+}
```

Comment:
Before unpinning, have we done a sanity check with a tgz build with the security ui, etc, to ensure we aren't hitting the Chrome crash? Double checking seems handy since it's caused an issue before.

Reply:
Not in an app, but I tested a reproducing case in a local Storybook build. I can also try it in the security ui to make sure nothing else wacky happens.

Reply:
No issues.